turnserver/src/controller/controller.go

```diff
@@ -0,0 +1,80 @@
+package controller
```

Comment:
is there a reason for the extra src dir?

Reply:
just for code structure